internal/mode/static/nginx/conf/nginx-plus.conf

```diff
@@ -26,6 +26,21 @@ http {
   sendfile on;
   tcp_nopush on;
 
+  server {
+    listen 8765;
```

Comment:
Chose this port to be easy to remember and avoid conflicts with common ports.

Was originally intending on making it customizable, however it lives in the bootstrap config right now, so we can't do that. Figured we can enhance this in the future to make this server configurable in the code instead of being in the bootstrap conf, but that may require a little bit of refactoring with current server template logic.

Reply:
> Was originally intending on making it customizable, however it lives in the bootstrap config right now, so we can't do that. Figured we can enhance this in the future to make this server configurable in the code instead of being in the bootstrap conf, but that may require a little bit of refactoring with current server template logic.

me thinks if customization will belong to a Custom resource that we haven't yet implemented :)

Reply:
Possibly